### ISSUE TITLE
Let's fail form labels that are set to display:none

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#62]
`<input>`  fields that have a `<label>` with a class that has css `display:none` don't read to screen readers...  consider a test to sniff for  that on an label. and fail it under 1.3.1

`<label class="hidden" for="searchText">search</label>`
CSS
`.hidden {
    display: none;
}`

**Comment by irwink** [comment]
Tool version 4.7.0 should detect and report this as an error.  Is the CSS on page or in a CSS style sheet file ? (my test page has it on page in a style tag).

**Comment by DavidMacDonald** [comment]
in the style page :(

**Comment by DavidMacDonald** [comment]
ps I didn't close it... github must of misinterpreted something

**Comment by DavidMacDonald** [comment]
I't in the style sheet

**Comment by irwink** [comment]
Do you have a sample page with this ?  I have a test page that uses external CSS and 'display: none'.  This test page works (H44 technique listed as failed).

**Comment by DavidMacDonald** [comment]
Will look into this when the crazy year end stuff ends...

**Comment by DavidMacDonald** [comment]
recently checked aria-describedby and it actually reads a describedby with display none... perhaps park this request for now... will follow up as time permits.

**Comment by DavidMacDonald** [comment]
I think this can be closed. aria-describedby reads label inside display:none.
